drf_spectacular/openapi.py

```diff
@@ -1016,6 +1023,47 @@ def _get_response_for_code(self, serializer, status_code, media_types=None):
             # https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#responseObject
             'description': ''
         }
+        if headers:
```

Comment:
its nitpicky sry, but move the `headers` definition above the `if`, and single quotes below. thanks!

Reply:
the double quotes are 100% force of habit since we use black for code formatting, will adapt